app/src/main/java/com/example/tasky/agenda_details/presentation/AgendaDetailsViewModel.kt

```diff
@@ -91,15 +93,122 @@ class AgendaDetailsViewModel @Inject constructor(
     }
 
     private fun loadDataForEvent() {
-        // todo
+        if (agendaItemId == null) {
+            throw IllegalArgumentException("agendaItemId is null")
+        }
```

Comment:
Couldn't this be null for a new agenda item or how do you check that?

Reply:
new agenda item does not call loadDataForEvent because its new :D